LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,306 @@
+    Licensed Software delivery as a convenience for development and testing
+    only. Customer acknowledges:
+
+    1. that some Third-Party Libraries may require additional licensing of
+        copyright and/or patents from the owners of such, and
+
+    2. that distribution of any of the Licensed Software referencing any portion
+        of a Third-Party Library may require appropriate licensing from such
+        third parties. All such Third Party Libraries, along with applicable
+        copyright notices and licenses, will be listed and made available to
+        Customer.
+
+3. As part of the license grant, a Slint Software License Certificate
+    (**License Certificate**) will be issued containing details of
+
+    1. the license grant,
+
+    2. the permitted scope of Customer's use of the Licensed Software, and
+
+    3. contact details of the Customer.
+
+    In case of any changes to the information provided in 3.2 and 3.3 above,
+    Customer is obliged to notify SixtyFPS in writing, not later than 7 days
+    after any such change has occured.
+
+4. The Term of this Agreement shall begin on the date mentioned in the License
+    Certificate (**Effective Date**) and shall continue unless terminated
+    earlier as herein provided (**Term**).
+
+5. New releases of the Licensed Software (**Updates**) will be made available
+    to Customer for the period (**Maintenance Period**) as specified in the
+    License Certificate. Such Updates may be bound by additional license terms.
+    Customer consents to any such additional license terms, without which
+    Customer may not install, copy, or otherwise use such Updates.
+
+6. The Licensed Software is to be used exclusively with Customer's software
+    products (**Customer's Products**) as specified in the License
+    Certificate.
+
+7. SixtyFPS grants Customer a perpetual, non-exclusive, non-transferable right
+    to use the Licensed Software, including making modifications to the Licensed
+    Software, during the Term. The number of users using the Licensed Software
+    is limited to the amount of **User Seats** as specified in the License
+    Certificate. For clarity, each user is counted as a User Seat.
+
+8. SixtyFPS grants Customer a perpetual, non-exclusive, non-transferable right
+    to reproduce and distribute the object code form of the Licensed Software,
+    including any modifications made to the Licensed Software, during the Term.
+    Such distributions is permitted only with Customer's Products and limited to
+    the amount of **Distributions** as specified in the License Certificate.
+
+9. The license grants are conditional on the full and timely payment of all
+    applicable Fees; if said Fees are not paid in full and by their due date,
+    the license shall not be granted and the Customer's Products shall be
+    deemed as made in breach of the Agreement ex tunc.
+
+10. Clause 10 above shall not apply for an **Ambassador License**, a license
```

Comment:
Fix clause reference. That said, we refer to an "Ambassador License" here - is that correct?

```suggestion
10. Clause 9 above shall not apply for an **Ambassador License**, a license
```